src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -2148,6 +2152,12 @@ - (void)downloadLogOfType:(MTRDiagnosticLogType)type
                                       completion:completion];
 }
 
+- (NSString *)description
```

Comment:
are we going to use this in a future PR? Don't see any usage in this one?

Reply:
It's used in this PR in the new log lines where we pass `self` as `%@`, no?

Reply:
I see. So self calls the description method. Didn't know that. Thanks.

Reply:
Converting an id to %@ will call `description` if it's an NSObject.